modules/xpfunmac.cmake

```diff
@@ -1032,8 +1032,10 @@ macro(xpSourceListAppend)
       list(SORT topdir) # sort list in-place alphabetically
       foreach(item ${topdir})
         xpGlobFiles(repoFiles ${item} *)
-        xpGlobFiles(fmtFiles ${item} *.c *.h *.cpp *.hpp *.cu *.cuh *.proto)
       endforeach()
+      set(fmtFiles ${repoFiles})
+      list(FILTER fmtFiles INCLUDE REGEX "^.*\.(c|h|cpp|hpp|cu|cuh|proto)$")
+      list(FILTER fmtFiles EXCLUDE REGEX "^.*\.crtoolrc$") # CMake was erroneously matching .crtoolrc
```

Comment:
Don't know what else to do about this one.  Maybe a cmake update might fix this?

Reply:
I'm fine with this, for now...